certified-connectors/Cyberday/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Cyberday
+Cyberday is an information security management system, where one key part is managing the data assets of a company. Often the most important data asset are the data systems used to process data and run operations.
+
+This connector allows you to fetch and create data systems inside your Cyberday account and integrate with
+other products in your organisation.
+
+## Publisher: Agendium Ltd
+
+## Prerequisites
+A Cyberday account is required to use this connector. Signing up can be done either at https://www.cyberday.ai/trial or directly through Microsoft Teams app store.
+
```

Comment:
Sections Known Issues and Limitations and Deployment Instructions are missing. Please follow the template provided on this [link](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors)